### PR TITLE
Bug 1163921 - [gaia-list] Avoid to qualify class rules with tag names for the element inside list

### PR DESCRIPTION
**Pull request by gasolin**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,18 +54,22 @@
       background: var(--background-plus);
     }
 
-    [dir=rtl] gaia-list.contacts .face {
+    gaia-list.contacts .gaia-item-circular > img {
+      width: 100%;
+    }
+
+    [dir=rtl] gaia-list.contacts .gaia-item-circular {
```
Comment (gasolin) [inline index.html]: moving into gaia-button with `:host-context([dir=rtl]) ::content .gaia-item-circular {` not work :(

File: gaia-list.js [hunk 2]
```diff
@@ -267,6 +271,7 @@ module.exports = component.register('gaia-list', {
     ::content .divided {
       -moz-border-start: solid 1px;
       -moz-padding-start: 14px;
+      border-radius: 0 !important;
```
Comment (gasolin) [inline gaia-list.js]: make divider not round (move hack from example page)

File: index.html (hunk 1 above)
Comment (wilsonpage) [inline index.html]: I don't understand. What's the problem?

Comment (gasolin) [inline index.html]: the style is not showing in related inline CSS style, so its might be related to `gaia-component` host-context hack

**Pull request merged**